examples/vectorized_representations/vectorized_representations.ipynb

```diff
@@ -1,20 +1,28 @@
 {
  "cells": [
   {
-   "cell_type": "code",
-   "execution_count": null,
-   "id": "e3695e71-18c0-4891-90da-f733f11af80c",
+   "cell_type": "markdown",
+   "id": "81faf0a0-198b-4f01-bbf5-1df93d58291a",
+   "metadata": {},
+   "source": [
+    "# Vectorized force field representations"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a5f5339e-78a4-44f0-a070-26e8efdc6d9b",
    "metadata": {},
-   "outputs": [],
    "source": [
-    "# Vectorized representations"
+    "Interchange can produce representations of both force field parameters and parametrized systems as NumPy arrays, which might be convenient for any number of computational approaches to force field development. This example explores this feature."
    ]
   },
```

Comment:
In this app, this is rendered like it's a title:

<img width="851" alt="image" src="https://github.com/openforcefield/openff-interchange/assets/7935382/785a9dcd-6dfb-43fe-8731-39504fe7df3e">


Though it looks fine to me in this file: https://github.com/openforcefield/openff-interchange/blob/13343b2df39f39e62fec54e4674872df2c572f09/examples/vectorized_representations/vectorized_representations.ipynb

Reply:
That's bizarre, they're not even in the same cell...

GitHub has rich Notebook diffs in a feature preview btw: https://github.blog/changelog/2023-03-01-feature-preview-rich-jupyter-notebook-diffs/